### DISCUSSION TITLE
Argon mania skin's special key adjustments

### DISCUSSION THREAD

**Discussion by Litttlefish** [ppy/osu#20624] [Ideas] [answered]
I've used lazer for a year or so (and only played the lazer version), and it's great to have a new skin, but the new wide middle key in 7k and 5k is a bit hard for me to get used to it since the old "Triangle" and "Classic" skin has even width for every key.

**Comment by peppy** [comment, marked as answer]
Thanks for the feedback. This will be customisable in the future.
